plugins/hls-tactics-plugin/src/Wingman/EmptyCase.hs

```diff
@@ -68,7 +68,9 @@ emptyCaseInteraction = Interaction $
             range = realSrcSpanToRange $ unTrack ss
         matches <-
           liftMaybe $
-            destructionFor
+          -- TODO(sandy): This needs a judgment in order to correctly compute
```

Comment:
Filed at #2173

Reply:
maybe it worths to add the link to the issue?